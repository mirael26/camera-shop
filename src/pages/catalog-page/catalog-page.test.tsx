import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import CatalogPage from './catalog-page';

jest.mock('../../components/header/header', () => 'Header');
jest.mock('../../components/promo/promo', () => 'Promo');
jest.mock('../../components/breadcrumbs/breadcrumbs', () => 'Breadcrumbs');
jest.mock('../../components/catalog/catalog', () => 'Catalog');
jest.mock('../../components/footer/footer', () => 'Footer');
jest.mock('../../components/modal/add-to-cart-modal/add-to-cart-modal', () => 'AddToCartModal');

test('CatalogPage renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<CatalogPage/>);
  expect(asFragment()).toMatchSnapshot();
});
