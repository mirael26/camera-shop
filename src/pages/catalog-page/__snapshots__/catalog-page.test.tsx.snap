// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CatalogPage renders correctly 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
    data-testid="catalog-page"
  >
    <header />
    <addtocartmodal />
    <main>
      <promo />
      <div
        class="page-content"
      >
        <breadcrumbs />
        <catalog />
      </div>
    </main>
    <footer />
  </div>
</DocumentFragment>
`;
